Tasks/ANTV1/task.json

```diff
@@ -17,7 +17,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 155,
+        "Minor": 157,
```

Comment:
The `Minor` version needs to be 158 for all tasks - as per our convention used to update the service :smile:

Reply:
Ok, that has been done !